sdk/core/core-http/src/policies/systemErrorRetryPolicy.ts

```diff
@@ -88,7 +88,8 @@ export class SystemErrorRetryPolicy extends BaseRequestPolicy {
   public sendRequest(request: WebResourceLike): Promise<HttpOperationResponse> {
     return this._nextPolicy
       .sendRequest(request.clone())
-      .then((response) => retry(this, request, response));
+      .then((response) => retry(this, request, response))
```

Comment:
do we need this line? I think the catch handles retrying on error

Reply:
Removed. I guess the original code was using `retry()` to return the response in the successful case...